From dad204699d870a86849a635ebec84f3259923a0e Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Thu, 4 Feb 2021 14:23:58 +0000
Subject: [PATCH] dt: Add option for dpi without DE and PCLK (for
 VGA666)

VGA666 doesn't use the DE or PCLK signals, therefore there is
no point in claiming their use. It's also then possible to
use GPIOs 0&1 for DDC to read the EDID from the display.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/bcm270x.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

--- a/arch/arm/boot/dts/bcm270x.dtsi
+++ b/arch/arm/boot/dts/bcm270x.dtsi
@@ -170,6 +170,12 @@
 			     20 21>;
 		brcm,function = <BCM2835_FSEL_ALT2>;
 	};
+	dpi_18bit_gpio2: dpi_18bit_gpio2 {
+		brcm,pins = <2 3 4 5 6 7 8 9 10 11
+			     12 13 14 15 16 17 18 19
+			     20 21>;
+		brcm,function = <BCM2835_FSEL_ALT2>;
+	};
 };
 
 &uart0 {
